openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -944,3 +994,22 @@ class nontaxable_ubi(Variable):
     entity = TaxUnit
     definition_period = YEAR
     documentation = """Amount of UBI benefit excluded from AGI"""
+
+
+class aftertax_income(Variable):
```

Comment:
similarly, aftertax_income and benefit_value_total aren't used for calculations. aftertax_income is the net income without benefits, and therefore used for summarizing tax reforms. Could be important for testing for now but I think eventually at least we may want to remove it.

Reply:
Yeah I agree, but I think it might still be helpful when validating the model